test/e2e/network/ingress.go

```diff
@@ -311,6 +311,42 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
 			executeBacksideBacksideHTTPSTest(f, jig, "")
 		})
 
+		It("should support multiple TLS certs [Unreleased]", func() {
+			By("Creating an ingress with no certs.")
+			jig.CreateIngress(filepath.Join(framework.IngressManifestPath, "multiple-certs"), ns, map[string]string{
```

Comment:
I gave this comment on one of @nikhiljindal's PR too. It would be great if the map and `filepath.Join()` are assigned to variables outside this call and then passed here. The current form is somewhat confusing.

Reply:
Will fix in a followup PR.